mmv1/products/bigqueryreservation/Reservation.yaml

```diff
@@ -58,8 +58,3 @@ properties:
     default_value: 0
     description: |
       Maximum number of queries that are allowed to run concurrently in this reservation. This is a soft limit due to asynchronous nature of the system and various optimizations for small queries. Default value is 0 which means that concurrency will be automatically set based on the reservation size.
-  - !ruby/object:Api::Type::Boolean
```

Comment:
even if advised from the bigquery team this is a breaking change that we'll need to separate out into a major version. Can we start by deprecating and giving the reason in the deprecation message?

Reply:
IIUC It hasn't been added yet, so it's not a breaking change to remove it

Reply:
oh, never mind I removed the wrong field 😳 I meant to pull this from the new resource.